src/authentication/views/registration-flow/r10-student-teacher.tsx

```diff
@@ -33,19 +32,19 @@ const StudentTeacher: FunctionComponent<StudentTeacherProps> = ({ history }) =>
 					/>
 				</MetaTags>
 				<div className="c-content">
-					<Button
-						type="secondary"
-						onClick={() =>
-							redirectToClientPage(APP_PATH.MANUAL_ACCESS_REQUEST.route, history)
-						}
-						icon="arrow-left"
-						title={t(
-							'authentication/views/registration-flow/r-10-student-teacher___ga-terug-naar-de-manuele-account-aanvraag-pagina'
-						)}
-						ariaLabel={t(
-							'authentication/views/registration-flow/r-10-student-teacher___ga-terug-naar-de-manuele-account-aanvraag-pagina'
-						)}
-					/>
+					<Spacer margin="bottom-large">
+						<Button
+							type="secondary"
+							onClick={() => history.goBack()}
```

Comment:
Kan je hier niet gewoon `onClick={history.goBack}`? Of heeft goBack optional params?

(Indien dit niet mogelijk is, resolve deze comment maar. :)